live-examples/css-examples/font-weight.html

```diff
@@ -1,35 +1,49 @@
-<section id="example-choice-list" class="example-choice-list" data-property="fontWeight">
-<div class="example-choice">
+<section id="example-choice-list" class="example-choice-list" data-property="font-family">
```

Comment:
We can drop the data attribute here as `font-family` is widely supported so we do not need a check before showing the examples.